src/pybind/fst/symbol_table_pybind_test.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
```

Comment:
Just a small detail -- wouldn't be better to have this as
```
#!/usr/bin/env python3
```
?

Reply:
Yes, I agree with you, but I am not sure whether kaldi pybind supports only Python3.

> Python 2.7 will not be maintained past 2020.

Is this a good time to switch to Python3 and to support only Python3 in the future for kaldi pybind?